mysys/crc32/crc32_arm64.c

```diff
@@ -19,17 +19,25 @@ int crc32_aarch64_available(void)
   return (auxv & HWCAP_CRC32) != 0;
 }
 
+int crc32c_aarch64_available(void)
+{
+  unsigned long auxv= getauxval(AT_HWCAP);
+  return (auxv & HWCAP_CRC32) != 0;
+}
+
```

Comment:
Please replace all references to `crc32c_aarch_available()` with `crc32_aarch64_available()`, and compute the `pmull_supported` flag in that function, using only one `getauxval()` call:
```c
const char *crc32c_aarch64_available(void)
{
  unsigned long auxv= getauxval(AT_HWCAP);
  if (!(auxv & HWCAP_CRC32))
    return NULL;
  pmull_supported= (auxv & HWCAP_PMULL) != 0;
  if (pmull_supported)
    return "Using ARMv8 crc32 + pmull instructions";
  else
    return "Using ARMv8 crc32 instructions";
}
```